includes/admin/class-wc-subscriptions-admin.php

```diff
@@ -1893,7 +1893,7 @@ public static function add_recurring_payment_gateway_information( $settings ) {
 
 		if ( ! $payment_gateways_handler::one_gateway_supports( 'subscriptions' ) ) {
 			// translators: $1-2: opening and closing tags of a link that takes to Woo marketplace / Stripe product page
-			$available_gateways_description = sprintf( __( 'No payment gateways capable of processing automatic subscription payments are enabled. If you would like to process automatic payments, we recommend the %1$sfree Stripe extension%2$s.', 'woocommerce-subscriptions' ), '<strong><a href="https://www.woocommerce.com/products/stripe/">', '</a></strong>' );
+			$available_gateways_description = sprintf( __( 'No %1$spayment gateways capable of processing automatic subscription payments%2$s are enabled. If you would like to process automatic payments, we recommend %3$sWooPayments%4$s.', 'woocommerce-subscriptions' ), '<a href="hhttps://woocommerce.com/document/subscriptions/payment-gateways/">', '</a>', '<strong><a href="https://woocommerce.com/payments/">', '</a></strong>' );
```

Comment:
I noticed this change was suggested on the issue, but I'm thinking this additional link ([payment gateways capable of processing automatic subscription payments](https://woocommerce.com/document/subscriptions/payment-gateways/)) is necessary.

IMO this section feels pretty crowded with links and since this href is already used further down, I think we can remove it. Thoughts?

![image](https://github.com/user-attachments/assets/4136b9be-4510-45c1-b7c6-585e60913015)

Reply:
Makes sense, removed on https://github.com/Automattic/woocommerce-subscriptions-core/pull/766/commits/b44e0b924d4768a07a37c3d4f2a6a97a2f0eb1de.